setup.py

```diff
@@ -35,6 +35,17 @@
 PY3 = sys.version_info[0] == 3
 
 
+# See LEGAL-362
+def verify_gpl_dependency():
+    if (not os.getenv("GPL_UNIDECODE")
+            and not os.getenv("SLUGIFY_USES_TEXT_UNIDECODE") == "yes"):
```

Comment:
Prefix with `AIRFLOW_`?

Reply:
can do

Reply:
Oh, though these vars are used by slugify's setup.py, I understand now. Right. Hmmmmm

Thinking and looking